src/structure/pure/Line.js

```diff
@@ -15,31 +15,47 @@
  * You should have received a copy of the GNU General Public License along
  * with this program; if not, write to the Free Software Foundation, Inc.,
  * 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA
- *
- * @module Line 
- * @title  CometVisu Line 
  */
 
 
 /**
+ * A line tag is used to display a horizontal line in the browser to allow for a grouped and thereby tidy display of
+ * elements on one page. To specify the width of the line an optional &lt;layout/&gt;-child can be added.
+ *
+ * @widget_example <meta selector="hr">
+ *   <caption>A line tag which uses 50% of the screen size (6 of 12 available columns)</caption>
+ * </meta>
+ * <line><layout colspan="6"/></line>
+ *
+ * @module structure/pure/Line
+ * @requires structure/pure
  * @author Christian Mayer
- * @since 2012
+ * @since 0.8.0 (2012)
```

Comment:
I would prefer to use the version the widget was introduced, as we might use this information for the user manual and we don't need the source file version there. But I don`t really know which widgets were there from the beginning (possibly all marked with 2012?)